mne/viz/evoked.py

```diff
@@ -1724,12 +1730,14 @@ def whitened_gfp(x, rank=None):
         )
     else:
         ax.legend(loc="upper right", prop=dict(size=10))
-        params = dict(
-            top=[0.69, 0.82, 0.87][n_rows - 1], bottom=[0.22, 0.13, 0.09][n_rows - 1]
-        )
-        if has_sss:
-            params["hspace"] = 0.49
-        fig.subplots_adjust(**params)
+        if not isinstance(fig.get_layout_engine(), ConstrainedLayoutEngine):
+            params = dict(
+                top=[0.69, 0.82, 0.87][n_rows - 1],
+                bottom=[0.22, 0.13, 0.09][n_rows - 1],
+            )
+            if has_sss:
+                params["hspace"] = 0.49
+            fig.subplots_adjust(**params)
```

Comment:
I think I'd prefer just to get rid of this stuff, and if people pass a fig with non-constrained-layout expect them to fix the layout issues
```suggestion
```